lib/ui/semantics.dart

```diff
@@ -585,7 +585,9 @@ class SemanticsUpdateBuilder extends NativeFieldWrapperClass2 {
     Float64List transform,
     Int32List childrenInTraversalOrder,
     Int32List childrenInHitTestOrder,
+    @Deprecated('use additionalActions instead')
```

Comment:
I would like to renamed this member - I plan on doing this in the next update but I want to make the roll process easier.  Happy to discuss an alternative approach.

Reply:
Seems like a good way to do it. Just make sure to roll the engine as soon as you submit to update the callsites in the framework.